src/components/profile/Profile.js

```diff
@@ -26,6 +28,32 @@ const Profile = () => {
       .catch((err) => console.error(err.response));
   }, [params]);
 
+  useEffect(() => {
+    client
+      .get('/cohort')
+      .then((res) => setCohortsAvailable(res.data.data))
+      .catch((err) => console.error(err.response));
+  }, [])
+
+  const displayMsgTwoSecs = () => {
+    setMsgIsDisplayed(true);
+
+    setTimeout(() => setMsgIsDisplayed(false), 2000);
```

Comment:
Normally I'd expect a requirement to describe this - try to stick exactly to the requirements

Reply:
What's the reasoning for this timeout?

Reply:
The timeout was to show a message to the user for 2 seconds, giving feedback on the status of the request.
We went ahead and deleted since it wasn't a requirement.